pdstemplate/__init__.py

```diff
@@ -1140,6 +1153,15 @@ def FILE_ZULU(filepath):
             form "yyyy-mm-ddThh:mm:ssZ".
         """
 
+        ##################################################################################
+        local_filepath = FCPath(filepath).retrieve()
+        # Only warn for remote files where cached time is returned
+        if local_filepath != filepath:
+            filepath = local_filepath
+            logger = get_logger()
+            logger.warning(f'FILE_TIME reflects the cached time: {filepath}', force=True)
+        ##################################################################################
+#        timestamp = FCPath(filepath).stat().st_mtime  ### not implemented in FCPath
```

Comment:
As discussed elsewhere, this should be `FCPath(filepath).modification_time()`, which will work today for both local and cloud storage locations.